test/vre/chapter36/config/applicantInformation.unit.spec.jsx

```diff
@@ -50,8 +58,9 @@ describe('VRE chapter 36 applicant information', () => {
     expect(form.find('.usa-input-error').length).to.equal(1);
     expect(onSubmit.called).to.be.false;
   });
+  */
 
-  it('should not submit without required applicant info', () => {
+  it.only('should not submit without required applicant info', () => {
```

Comment:
Need some guidance on what's being tested here and how to set it up in the form. Is input from required fields necessary before **Continue** or before **Submit** on the review page? Currently I can continue from this page with none of the questions answered.

Reply:
According to the design, the questions on this page aren't required, I'll invite @jbalboni and @stacyhlinn to chime in as well.